### DISCUSSION TITLE
Wrenches?

### DISCUSSION THREAD

**Discussion by pat1121** [aderusha/DDD-Printable-Wall-Control-System#31] [Q&A]
I see them mounted in your demonstration picture, but can't figure out which files these are? Can anyone point me in the right direction? Thanks!

**Comment by aderusha** [comment]
Maybe [this](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Centerpieces/Tool_hooks/1x6%2011pc%20Wrench%20Hanger.stl) or [this](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Centerpieces/Tool_hooks/1x6%2011pc%20Wrench%20Hanger%20Inverted.stl)?

**Reply by boasist** [reply]
I found the files you linked to here, but what are the other blue parts used to create the whole unit? I'm having trouble dialing those in. Thank you.

**Comment by boasist** [comment]
You need to use Centerpieces/Spacer_clip-on/4x10x8mm Pin.stl to connect the wrench holders to the center, left, and right mounts and then all is good. That little pin is pretty crucial to everything.

**Reply by dylo** [reply]
Thanks I'm still struggling with this one trying to figure out the parts shown in the photo.  I can find the two ones labeled wrench referenced above but then it looks like I print 2 of those and there's still a piece in between them and a piece on either end.  How does that little pin that's essentially just a block work?  Can someone list out all the pieces required?  Sorry this is my first one.

**Reply by boasist** [reply]
Sure - 

All parts available [here](https://www.printables.com/model/52618-ddd-printable-wall-control-system/files)

x2 Wrench hanger - 1x6-11pc-wrench-hanger.stl
Spacer - 1x6-spacer-blank.stl - I did a 1x6 yet you can do a 2x6 or larger depending on your wrenches
x24 Connecting Pins - 4x10x8mm-pin.stl
x2 Hangers - 6x0-flat-center.stl

The connecting pins will go in all 6 holes on one side of each part, clamp or push the parts together, then mount and enjoy.

There is also the "inverted" wrench holder if you want your large wrenches up top and small on bottom, As seen in the photos.

**Reply by dylo** [reply]
Should I be using two of these instead of that center spacer?  It has notches already on it that would fit perfectly.  I don't see a 6x1 that would fit or I don't know how to use flush one maybe

3x1-spacer-blank.stl

**Comment by dylo** [comment]
Is this the right center piece?  I feel like I need one with a place for inserts on the sides not just one on each end.  Or am I doing this wrong.

![IMG_6178](https://github.com/aderusha/DDD-Printable-Wall-Control-System/assets/10145399/3a599b3b-0038-4fbc-a0b5-d3b72b22f855)

**Comment by aderusha** [comment]
# Here's a complete recipe

2x [1x6 11pc Wrench Hanger.stl](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Centerpieces/Tool_hooks/1x6%2011pc%20Wrench%20Hanger.stl)
1x [6x0 Flat Center.stl](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Sidepieces/Flats/6x0%20Flat%20Center.stl)
1x [6x0 Flat Left.stl](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Sidepieces/Flats/6x0%20Flat%20Left.stl)
1x [6x0 Flat Right.stl](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Sidepieces/Flats/6x0%20Flat%20Right.stl)
24x [4x10x8mm Pin.stl](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Accessories/4x10x8mm%20Pin.stl)

**Reply by dylo** [reply]
Ah now it's making sense.  The blank center was throwing me off.  I'm going to print the proper side pieces now and I should be good thank you!

**Comment by dylo** [comment]
Came together nicely I think!

![IMG_6183](https://github.com/aderusha/DDD-Printable-Wall-Control-System/assets/10145399/712b2e10-bc65-448b-a41d-27278f6c3b3d)
![IMG_6184](https://github.com/aderusha/DDD-Printable-Wall-Control-System/assets/10145399/1e9d14d7-141d-4ce0-a9a2-bb5bfc2a571d)

**Comment by joec2000** [comment]
Sorry for resurrecting an old discussion, but is there a way to make one of these fit thicker wrenches? I have hundreds of wrenches and currently, only my GearWrench racheting wrenches fit these. My Tekton and Craftsman raised panel wrenches are all too thick. Is there a way to either produce a larger version of this wrench holder, designed for thicker wrenches, or more ideally, parameterize it in some way to properly scale the part (and side pieces) to hold thicker wrenches?

I don't know much about STL files, but it would be SUPER cool to see a script of some kind that asked you how many wrenches you wanted to hold, how thick they were, etc and it just spat out properly scaled STL files :)

**Reply by boasist** [reply]
@joec2000 Depending on how much you need to trim, it'd be pretty easy to modify this in tinkercad by creating a hollow piece to trim out the holders, without needing any modifications to the mounts.

Beyond that, you're talking custom work.
